### PR TITLE
RedshiftUsingCloudwatchPublish

### PR DESCRIPTION
**Pull request by apurva-birajdar**
JIRA case [GRD-76410]: Publish cloudwatch for AWS Redshift

### CODE REVIEW

**Review by RefaelAdi — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]

**Review by chirag-ibm — APPROVED** [review]

File: filter-plugin/logstash-filter-redshift-aws-guardium/README.md [hunk 1]
```diff
@@ -97,6 +97,14 @@ Go to the S3 buckets from the search box and find the details of the generated l
 
 `s3`/<`bucket`>/<`prefix/`>/`AWSLogs/`/<`Account ID`>`redshift/`/<`region/`>/`<Year/>`/`<Month/>`/`<Day/>`/ See the generated UserActivity/Connection logs here.
 
+## Viewing the logs entries on Cloudwatch
+
+### Procedure
+Go to Cloudwatch from the search box and find the details of the generated logs (UserActivity/Connection) in below log groups:
```
Comment (mllezak) [inline filter-plugin/logstash-filter-redshift-aws-guardium/README.md:103]: Change: 
)  in below log groups:

To:

) in the following log groups:

File: filter-plugin/logstash-filter-redshift-aws-guardium/README.md [hunk 2]
```diff
@@ -112,16 +120,21 @@ The Guardium universal connector is the Guardium entry point for native audit lo
...
-6. Update the input section to add the details from [redshift.conf](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-redshift-aws-guardium/redshift.conf) file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
-7. Update the filter section to add the details from [redshift.conf](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-redshift-aws-guardium/redshift.conf) file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end.
+6.  If the audit logs are to be fetched from s3 directly, use the details from the [redshift-over-s3.conf](redshift-over-s3.conf)
+    file, If the audit logs are to be fetched from Cloudwatch use the details from [redshift-over-cloudwatch.conf](redshift-over-cloudwatch.conf) file.Update the input section to add the details
+    from the corresponding file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
+7.  If the audit logs are to be fetched from s3 directly, use the details from the [redshift-over-s3.conf](redshift-over-s3.conf)
```
Comment (mllezak) [inline filter-plugin/logstash-filter-redshift-aws-guardium/README.md:135]: I think step 7 is a duplicate of step 6. Delete.

**Review by mllezak — APPROVED** [review]
I changed a couple of things.  Consider this approved with changes.

File: filter-plugin/logstash-filter-redshift-aws-guardium/README.md (hunk 1 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-redshift-aws-guardium/README.md:103]: done

File: filter-plugin/logstash-filter-redshift-aws-guardium/README.md [hunk 3]
```diff
@@ -112,16 +120,21 @@ The Guardium universal connector is the Guardium entry point for native audit lo
...
 4. Click the Plus sign to open the Connector Configuration dialog box.
 5. Type a name in the **Connector name** field.
-6. Update the input section to add the details from [redshift.conf](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-redshift-aws-guardium/redshift.conf) file's input part, omitting the keyword "input{" at the beginning and its corresponding "}" at the end.
-7. Update the filter section to add the details from [redshift.conf](https://github.com/IBM/universal-connectors/raw/main/filter-plugin/logstash-filter-redshift-aws-guardium/redshift.conf) file's filter part, omitting the keyword "filter{" at the beginning and its corresponding "}" at the end.
+6.  If the audit logs are to be fetched from s3 directly, use the details from the [redshift-over-s3.conf](redshift-over-s3.conf)
```
[the remark this answers is not in the document]
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-redshift-aws-guardium/README.md:132]: done.

File: filter-plugin/logstash-filter-redshift-aws-guardium/README.md (hunk 2 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-redshift-aws-guardium/README.md:135]: 2 lines are written . One is for input and another is for filter.

**Pull request merged**